pipeline-runner/start-batch.sh

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+python3 launch-datadog.py
```

to make this run in the background just do `python3 launch-datadog.py &`.

---

in case it's killed, use `nohup python3 launch-datadog.py &` but I am pretty sure it's not needed. `&` tells the process to run in the background (so next command will be run). `nohup` means, don't send any hang up signal to the process if the parent dies.

---

add the `#!/bin/bash -e` to the shebang, that way if we break the `launch-datadog.py` script we will notice because the pipelines won't run instead of failing silently

---

added suggestion